<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilteredTabelView</class>
 <widget class="QWidget" name="FilteredTabelView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_filter">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_limit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="placeholderText">
        <string>limit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>1/1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonL">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::LeftArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonR">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::RightArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="date">
      <date>
       <year>2021</year>
       <month>2</month>
       <day>3</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd hh:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="dateTimeEdit_2">
     <property name="date">
      <date>
       <year>2021</year>
       <month>12</month>
       <day>30</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd hh:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
